account_check_printing_report_base/README.rst

```diff
@@ -0,0 +1,109 @@
+==================================
+Account Check Printing Report Base
+==================================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Faccount--payment-lightgray.png?logo=github
+    :target: https://github.com/OCA/account-payment/tree/12.0/account_check_printing_report_base
+    :alt: OCA/account-payment
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/account-payment-12-0/account-payment-12-0-account_check_printing_report_base
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/96/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module provides the basic framework for check printing, and a sample
+layout.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Installation
+============
+
+In order to install this module you must first install also the module
+'report_paper_wkhtmltopdf_params', available in
```

Comment:
I'm not sure, but if you need to install this module first, shouldn't it be added as a dependency in the manifest?